src/foundry/foundry.js/cameraPopoutAppWrapper.d.ts

```diff
@@ -25,16 +25,20 @@ declare class CameraPopoutAppWrapper {
    */
```

Comment:
Nitpicking: There are some missing spaces at the constructor param description:
```
   * @param view    - The CameraViews application that this popout belongs to
   * @param userId  - ID of the user this popout belongs to
   * @param element - The div element of this specific popout window
```

Reply:
done